lib/config/defaults.js

```diff
@@ -289,7 +289,6 @@ const applyExperimentsDefaults = (
 ) => {
 	D(experiments, "futureDefaults", false);
 	D(experiments, "backCompat", !experiments.futureDefaults);
-	D(experiments, "topLevelAwait", experiments.futureDefaults);
```

Comment:
Just improve this:
- when `experiments.futureDefaults` it should be `true`
- when no valud, it should be `true`
- when `false`, it should be false